pkg/machinery/nethelpers/std_netaddr.go

```diff
@@ -6,24 +6,25 @@ package nethelpers
 
 import (
 	"net"
+	"net/netip"
 
-	"inet.af/netaddr"
```

Comment:
nit: this file might be renamed to `std_netip.go`

Reply:
Done!